optimum/onnxruntime/modeling_ort.py

```diff
@@ -193,11 +204,12 @@ def _from_pretrained(
                 kwargs will be passed to the model during initialization
         """
         config_dict = kwargs.pop("config", {})
+        provider = kwargs.pop("provider", None)
```

Comment:
we should add named argument to the method preferably with a type and a default value